var/spack/repos/builtin/packages/awscrt/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Awscrt(PythonPackage):
```

Comment:
From what I can tell, this is a Python library, not a command-line tool, so the package should be named `py-awscrt`

Reply:
Done